### DISCUSSION TITLE
storybook-addon depends on react

### DISCUSSION THREAD

**Discussion by iambumblehead** [urql-graphql/urql#2601] [Q&A] [answered]
[@urql/storybook-addon][1] depends on '[urql][0]' and thus, depends on react and is not usable w vue3

the storybook-addon causes storybook and vue3 startup to fail
```console
ModuleNotFoundError: Module not found: Error: Can't resolve 'urql' in
  '/root/app/storybook/node_modules/@urql/storybook-addon/dist'
```

[0]: https://www.npmjs.com/package/urql
[1]: https://www.npmjs.com/package/@urql/storybook-addon

**Comment by JoviDeCroock** [comment, marked as answer]
Will be solved by #2646
